lib/modules/manager/npm/post-update/index.ts

```diff
@@ -322,6 +322,27 @@ async function resetNpmrcContent(
   }
 }
 
+export function fuzzyMatchAdditionalYarnrcYml<
```

Comment:
sorry, but one last thing. this new function should be moved to `./yarn.ts`.

Reply:
Done, but as index.spec.ts mocks the yarn.ts import I removed some of the tests from index.spec.ts. Coverage is still there, but tests no longer verify the resultant yarnrc file to the same extent